5.bindings/kafka-non-persistence.yaml

```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+# Install 1 replica for e2e test
```

Comment:
nit: I think this e2e test can be removed..

Reply:
fixed.